### PR TITLE
Update compliance-manager-assessments.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Compliance manager assessors to the note as they have access to Build and manage assessment.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4474

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/compliance-manager-assessments.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ There are three ways you can set up assessments:
 3. [Create your own custom assessment](#create-your-own-custom-assessment).
 
 > [!NOTE]
-> Only users who hold a Global Administrator or  Compliance Manager Administration role can create and modify assessments. Learn more about [roles and permissions](compliance-manager-setup.md#set-user-permissions-and-assign-roles).
+> Only users who hold a Global Administrator or Compliance Manager Administration role can manage template creation and modification. The Compliance Manager Assessors role can create and modify assessments using templates created or modified by Global Administrator or Compliance Manager Administration. Learn more about [roles and permissions](compliance-manager-setup.md#set-user-permissions-and-assign-roles).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/compliance-manager-assessments.md]:
```suggestion
> Only users who hold a Global Administrator or Compliance Manager Administration role can manage template creation and modification. The Compliance Manager Assessors role can create and modify assessments using templates created or modified by the Global Administrator or Compliance Manager Administration. Learn more about [roles and permissions](compliance-manager-setup.md#set-user-permissions-and-assign-roles).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chvukosw** [comment]
The suggested edit isn't fully accurate per engineering. Will update this article in a separate PR to indicate that CM Assessors can also create and modify assessments.

**Pull request closed without merging**